apps/synapse-portal-framework/src/components/elportal/ELContributeYourData.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Link, Button } from '@mui/material'
+import React from 'react'
+import backgroundSpotsLeft from './assets/dot_blob_top_left.png'
+import backgroundSpotsRight from './assets/dot_blob_bottom_right.png'
+
+const ELContributeYourData: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: '#5BA998',
+        height: '250px',
+        textAlign: 'left',
+        display: 'flex',
+        alignItems: 'center',
+        position: 'relative',
+        overflow: 'hidden',
+      }}
+    >
+      <Box
+        sx={{
+          flex: '0 0 auto',
+          width: '33%',
+          height: '100%',
+          backgroundImage: `url(${backgroundSpotsLeft})`,
+          backgroundRepeat: 'no-repeat',
+          backgroundPosition: 'top left',
+          backgroundSize: '100%',
+          zIndex: 0,
+          overflow: 'clip',
+        }}
+      />
+      <Box
+        sx={{
+          flex: '1 1 auto',
+          zIndex: 1,
+          textAlign: 'center',
+          paddingTop: '100px',
+          paddingBottom: '100px',
+        }}
+      >
+        <Box
+          sx={{
+            width: '90%',
+            height: '3px',
+            backgroundColor: 'white',
+            opacity: 0.5,
+            marginBottom: 2,
+            margin: '0 auto',
+          }}
+        />
+        <h1 style={{ color: 'white', fontFamily: 'serif' }}>
+          Contribute Your Data
+        </h1>
+        <p style={{ color: 'white' }}>
+          If you are a funded portal contributor and ready to upload data to the
+          ELITE Portal, you can begin the data submission process by contacting
+          our data curation team through our service desk.
+        </p>
+        <Link
+          href="https://sagebionetworks.jira.com/servicedesk/customer/portal/12"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Button
+            variant="contained"
+            sx={{
+              backgroundColor: '#5BA998',
+              border: '1px solid white',
+              boxShadow: 'none',
+              '&:hover': {
+                backgroundColor: '#5BA998',
```

Comment:
For this custom hover color, slightly modify the base color so it feels 'interactive'. You can import `darken` from @material/mui - feel free to tweak the second argument as desired
```suggestion
                backgroundColor: darken('#5BA998', 0.05),
```